metricbeat/module/logstash/node_stats/data_xpack.go

```diff
@@ -96,7 +96,11 @@ type nodeInfo struct {
 	Status      string   `json:"status"`
 	HTTPAddress string   `json:"http_address"`
 	Pipeline    pipeline `json:"pipeline"`
-	Monitoring  struct {
+}
+
+type inNodeInfo struct {
```

Comment:
maybe adding some comments here to explain the rational behind this "wrapping" approach would be of benefit for people visiting this in the future.